### DISCUSSION TITLE
Welcome to jiofiunlocked Discussions!

### DISCUSSION THREAD

**Discussion by uc-priyadmm** [uc-priyadmm/jiofiunlocked#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by MiNote42k** [comment]
Hello Sir i flashed ur built jiofi jmr541 on my device it worked like Magic i tested with all sims it worked flawless...thanx for the file..
1 more request can u plz support calling for other operators via jio4gvoice app or any app that canbe used for calling..thanx in advance..

**Reply by uc-priyadmm** [reply]
Hey Boss! Let me try

**Comment by posttomanish** [comment]
Hi, I want to use my Jiofi3 JMR540 for other operator SIM. How can I crack it? Please help me.
